backend/models/ticket.js

```diff
@@ -231,11 +231,33 @@ class Ticket {
 		const systemComments = [];
 		const fields = Object.keys(data);
 
+		let newViewpoint;
+
+		if (data.viewpoint) {
```

Comment:
shouldn't this just live inside L241?

Reply:
@carmenfan I tried to have it on L251 and it didn't seem to wait for the screenshot to be created before proceeding